README.md

```diff
@@ -1,45 +1,57 @@
 # Interview Assignment - Coop Logistics API
 
-Before jump in:
+**This repository hosts the source code for a client designed to generate a
+specified number of requests (N).**
+
+**As part of the assignment, the candidate is
+required to develop an API server capable of handling these requests.
+The server can be implemented using either gRPC or HTTP protocols.**
+
+**The primary objective of this test is to assess the candidate's ability to
+create a robust API server that efficiently processes incoming requests as
+specified by the client's operations**
+
+Before jump in some usefully links:
 
 - [Golang documentation](https://go.dev/doc/)
 - [gRPC documentation](https://grpc.io/docs/)
 - [Buf tool to manage Protobuf files](https://buf.build/docs/introduction/)
 
-The test assignment for a Software Engineer
+The test assignment instructions for a Software Engineer
 -> [instructions](docs/instructions.md).
 
 ## Overview
 
 ![project_overview](docs/assets/overview.png)
 
-
 ## Repository Overview
 
 ### Folders
 
 ```text
-├── api                                  // API definition with gRPC proto-files
-├── docker-compose                       // Docker related files
-├── docs                                 // Instructions
-└── src                                  // Source code related to test assignment
-    ├── client                           // Client that will send requests to API Server
-    ├── generated                        // Shared code that can be used to implement gRPC
-    │   └── logistics
-    │       └── api
-    │           └── v1                   // protoc-gen-go of API
-    └── server                           // Your solution that provides API server
```

Comment:
minor: maybe also include this. `server   // Your solution that provides API server`